tokens/colors.json

```diff
@@ -0,0 +1,76 @@
+{
+  "global": {
+    "type": "color",
+    "category": "background-color"
+  },
+  "props": {
+    "COLOR_BRAND_WHITE": {
+      "value": "#FFFFFFF",
+      "comment": "Brand color white"
+    },
+    "COLOR_BRAND_GREY-LIGHT": {
+      "value": "#F5F6F9",
+      "comment": "Brand color grey light"
+    },
+    "COLOR_BRAND_GREY": {
+      "value": "#C9CFDD",
+      "comment": "Brand color grey"
+    },
+    "COLOR_BRAND_DARK": {
+      "value": "#6B7B96",
+      "comment": "Brand color dark"
+    },
+    "COLOR_BRAND_DARKEST": {
+      "value": "#011940",
+      "comment": "Brand color darkest"
+    },
+    "COLOR_BRAND_GREEN_LIGHT": {
+      "value": "#CEF1E4",
+      "comment": "Brand color green light"
+    },
+    "COLOR_BRAND_GREEN": {
+      "value": "#09BA76",
+      "comment": "Brand color green"
+    },
+    "COLOR_BRAND_GREEN_DARK": {
+      "value": "#069961",
+      "comment": "Brand color green_dark"
+    },
+    "COLOR_THEME_Green": {
```

Comment:
We are not consitent in using casing. here we use lowercase, but in others we use only Uppercase. Can we pick either one? My preference would be uppercase